modules/shadertools/src/lib/assemble-shaders.js

```diff
@@ -81,6 +81,8 @@ function assembleShader(
     glslVersion = 300; // TODO - regexp that matches atual version number
```

Comment:
@1chandu We really need to fix this TODO...

Reply:
Can we just update in this PR, since it's changing the assumption that `#version` is either not specified or 300. I think the following should work `glslVersion = sourceLines[0].match(/\b\d+\b/)[0];`

This will also be useful if we want to use `assembleShaders` WebGL compute or WebGPU...